### ISSUE TITLE
Cascading liquidation violates whitepaper

### ISSUE THREAD

**Issue by Sam202120201** [VenusProtocol/venus-protocol#48]
Venus liquidated me 95% on 19th May 2021 in the few minutes of market flash crash (not talking about XVS dump prior to that) , while the white paper says 50%. I need the excess liquidated money back. I need this to be reasonably solved as proof that the protocol cares about its users and doesn't take unfair advantage. 

Specific Issue:
I had 141.2749 BNB and  397.7783 XVS in my Venus deposit before the flash crash in markets on 19th May. At the crash time the Binance smart chain was stalled out and there was no chance given to do any form of repayment. I thought 50% liquidation risk is there as per the Venus white paper but then I saw my balance was turned into 13.1846 BNB and 12.3187 XVS in minutes !! That is more than 90% total liquidation and definitely and exploit in the system which unfairly benefits liquidator or Venus itself at the cost if the investor. 

This is the liquidiation guide extract on Venus: 
"How much do they liquidate us?
They liquidate us enough to pay our loan + 10% on that value that remains of profit for the liquidator.
Do they liquidate our entire guarantee?
No, the liquidator can pay up to a maximum of 50% of the value of the loan, paying us the corresponding value in our deposited currency plus its added 10%."

Othe protocols like Cream finance did not suffer any such serial cascading liquidation. Only Venus. 

Please refund me the extra amount liquidated as per the rules and in fairness. I have lost extraordinary money due to this flaw! I am totally shocked and heartbroken and I had such high expectations but now I have faced an unfathomable and unfair loss. A person can claw back from 50% but not from 90%. Please help me and do justice I had great faith in CZ and Joselito and the bsc ecosystem as 'for the people', so this must be investigated.

**Comment by whitematrix62** [comment]
VIP 19 was introduced to facilitate the cascading liquidations. The liquidation penalty was increased from 10% to 15%.

**Comment by Sam202120201** [comment]
But that's just daylight robbery. It should be mentioned that 'Venus will take all your money' if you put it here. How can this be justified? Your dapp has ruined many people , where Cream finance and other  similar ones did not. How do you justify claiming 95% of a user's capital?? I will start social media awareness against Venus unless Joselito homself says that Venus will compensate the users for stealing all their money. I can accept 50% liquidation as that was in the white paper. Not any more than that it's not fair at all!
________________________________

**Comment by Sam202120201** [comment]
Yes I would love to have 1 person from the team respond. Have tried twitter, telegram..no avail yet.

I thought you were one from the team, sorry.
________________________________

**Comment by whitematrix62** [comment]
But your loss is probably less than 95% as you did borrow some assets right?

**Comment by Sam202120201** [comment]
I had re deposited what I had borrowed. So I lost everything.
________________________________

**Comment by Sam202120201** [comment]
And cream finance liquidated only 15% and stopped , giving me a chance to repay. Venus kept liquidating and in bsc was slow even with 15 gwei gas. In few minutes venus liquidated me 3 times and ended everything. It is not fair at all.
________________________________

**Comment by whitematrix62** [comment]
Sorry to hear that. It is very puzzling why the team would push for vip19. It took more money than necessary out of the protocol, resulting in greater loss for the user and at the same time put the protocol at greater risk of being undercollateralized.

**Comment by Sam202120201** [comment]
Absolutely. It is self destructive. And there are other dapps to take cue from, who may have made mistakes in past. So why?  If the community is bankrupt, Venus is lost.
________________________________

**Comment by mbenatti** [comment]
> And cream finance liquidated only 15% and stopped , giving me a chance to repay. Venus kept liquidating and in bsc was slow even with 15 gwei gas. In few minutes venus liquidated me 3 times and ended everything. It is not fair at all.
> ________________________________
> From: Sameer Sahay ***@***.***>
> Sent: Tuesday, May 25, 2021 6:49:17 PM
> To: VenusProtocol/venus-protocol ***@***.***>; VenusProtocol/venus-protocol ***@***.***>
> Cc: Author ***@***.***>
> Subject: Re: [VenusProtocol/venus-protocol] Cascading liquidation violates whitepaper (#48)
> 
> I had re deposited what I had borrowed. So I lost everything.
> ________________________________
> From: whitematrix62 ***@***.***>
> Sent: Tuesday, May 25, 2021 6:30:43 PM
> To: VenusProtocol/venus-protocol ***@***.***>
> Cc: Sam202120201 ***@***.***>; Author ***@***.***>
> Subject: Re: [VenusProtocol/venus-protocol] Cascading liquidation violates whitepaper (#48)
> 
> 
> But your loss is probably less than 95% as you did borrow some assets right?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FVenusProtocol%2Fvenus-protocol%2Fissues%2F48%23issuecomment-847848339&data=04%7C01%7C%7C065cf22f617d4c0d29fe08d91f7d1bcd%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637575444477977373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=qinz33ePnnNUPAt%2BhJgAY0YcQ474GpaiOV98NuYF1dY%3D&reserved=0>, or unsubscribe<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAUGUQVFG2HX3PUIVVLDZSMDTPONPXANCNFSM45OKZRNQ&data=04%7C01%7C%7C065cf22f617d4c0d29fe08d91f7d1bcd%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637575444477977373%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=XZR%2FCmWN%2F5CzrgAMxVswsCH5UCbZSaiKLL%2FBEN%2BdOmQ%3D&reserved=0>.
> 

I had the same situation on BTC dump to 30k, cascading liquidations, losing 95% of my funds

**Comment by Skytokenn** [comment]
Kindly open this live chat link to talk to the customer service directly on your issue  https://direct.lc.chat/14334663/
